data/json/items/gun/762x25.json

```diff
@@ -33,9 +35,9 @@
     "id": "tokarev",
     "copy-from": "pistol_base",
     "type": "GUN",
-    "name": "Tokarev TT-30",
-    "name_plural": "Tokarev TT-30",
-    "description": "The Norinco manufactured Tokarev TT-30 is the standard sidearm of the Chinese military, it does not see extensive use outside of China.",
+    "name": "Tokarev TT-33",
+    "name_plural": "Tokarev TT-33",
+    "description": "The Norinco manufactured Tokarev TT-33 was the standard sidearm of the Chinese military, it does not see extensive use outside of China.",
```

Comment:
I know. I want to split the Tokarev into a Russian antique and a Chinese import version, but I don't think it will be accepted.